hydromt/models/model_lumped.py

```diff
@@ -153,56 +183,67 @@ def __init__(
             data_libs=data_libs,
             logger=logger,
         )
-
-    def read(
+    
+    def setup_response_unit(
         self,
-        components: List = [
-            "config",
-            "response_units",
-            "geoms",
-            "forcing",
-            "states",
-            "results",
-        ],
-    ) -> None:
-        """Read the complete model schematization and configuration from model files.
-
-        Parameters
-        ----------
-        components : List, optional
-            List of model components to read, each should have an associated read_<component> method.
-            By default ['config', 'maps', 'response_units', 'geoms', 'forcing', 'states', 'results']
+        split_regions = False,
+        split_method = "us_area",
+        hydrography_fn = "merit_hydro",
+        **kwargs
+        ):
+        """
+        This component sets up the region and response units.
+        
         """
-        super().read(components=components)
 
-    def write(
+        if len(self.region) == 0:
+            raise ValueError("No region defined. Define :py:meth:`~hydromt.models.model_api.setup_region`")
```

Comment:
Links to methods don't work in error messages, this is sphinxs docstrings specific I believe.

Reply:
Changed to "No region defined. Define region first with setup_region."